internal/pkg/cli/project_init.go

```diff
@@ -135,11 +134,7 @@ func (o *initProjectOpts) Execute() error {
 		Domain:    o.DomainName,
 	})
 	if err != nil {
-		// If the project already exists, move on - otherwise return the error.
```

Comment:
That's a good question! Never thought about the case when more then one ppl working on the same project. I guess in the scenario you put, when PH want to create a demo project, if they are using the same AWS account, the CLI should do a check first if it exists in the SSM already. This can also be applied to `env init`/`app init`.